worlds/overcooked2/Items.py

```diff
@@ -0,0 +1,156 @@
+from BaseClasses import Item
+from typing import Optional, NamedTuple, Dict
+import codecs
+
+
+class ItemData(NamedTuple):
+    code: Optional[int]
+
+
+class Overcooked2Item(Item):
+    game: str = "Overcooked! 2"
+
+
+oc2_base_id: int = int.from_bytes(codecs.encode(b"oc2", "hex"), byteorder="big")
+
+item_table: Dict[str, ItemData] = {
+    "Wood"                          : ItemData(oc2_base_id + 1 ),
+    "Coal Bucket"                   : ItemData(oc2_base_id + 2 ),
+    "Spare Plate"                   : ItemData(oc2_base_id + 3 ),
+    "Fire Extinguisher"             : ItemData(oc2_base_id + 4 ),
+    "Bellows"                       : ItemData(oc2_base_id + 5 ),
+    "Clean Dishes"                  : ItemData(oc2_base_id + 6 ),
+    "Larger Tip Jar"                : ItemData(oc2_base_id + 7 ),
+    "Progressive Dash"              : ItemData(oc2_base_id + 8 ),
+    "Progressive Throw/Catch"       : ItemData(oc2_base_id + 9 ),
+    "Coin Purse"                    : ItemData(oc2_base_id + 10),
+    "Remote Control Batteries"      : ItemData(oc2_base_id + 11),
+    "Wok Wheels"                    : ItemData(oc2_base_id + 12),
+    "Dish Scrubber"                 : ItemData(oc2_base_id + 13),
+    "Burn Leniency"                 : ItemData(oc2_base_id + 14),
+    "Sharp Knife"                   : ItemData(oc2_base_id + 15),
+    "Order Lookahead"               : ItemData(oc2_base_id + 16),
+    "Lightweight Backpack"          : ItemData(oc2_base_id + 17),
+    "Faster Respawn Time"           : ItemData(oc2_base_id + 18),
+    "Faster Condiment/Drink Switch" : ItemData(oc2_base_id + 19),
+    "Guest Patience"                : ItemData(oc2_base_id + 20),
+    "Kevin-1"                       : ItemData(oc2_base_id + 21),
+    "Kevin-2"                       : ItemData(oc2_base_id + 22),
+    "Kevin-3"                       : ItemData(oc2_base_id + 23),
+    "Kevin-4"                       : ItemData(oc2_base_id + 24),
+    "Kevin-5"                       : ItemData(oc2_base_id + 25),
+    "Kevin-6"                       : ItemData(oc2_base_id + 26),
+    "Kevin-7"                       : ItemData(oc2_base_id + 27),
+    "Kevin-8"                       : ItemData(oc2_base_id + 28),
+    "Cooking Emote"                 : ItemData(oc2_base_id + 29),
+    "Curse Emote"                   : ItemData(oc2_base_id + 30),
+    "Serving Emote"                 : ItemData(oc2_base_id + 31),
+    "Preparing Emote"               : ItemData(oc2_base_id + 32),
+    "Washing Up Emote"              : ItemData(oc2_base_id + 33),
+    "Ok Emote"                      : ItemData(oc2_base_id + 34),
+    "Ramp Button"                   : ItemData(oc2_base_id + 35),
+    "Bonus Star"                    : ItemData(oc2_base_id + 36),
+    "Calmer Unbread"                : ItemData(oc2_base_id + 37),
+}
+
+# exclusive
+oc2_end_id = item_table["Calmer Unbread"].code + 1
```

Comment:
I don't see this being used outside of tests, so i argue this should be moved to the test (i.e. expect "Calmer Unbread" to be the last item), not sure how useful that test is then though, but checking an unused variable does not test anything either.